### PR TITLE
[SD-2960] - Persisting the crop coordinates

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: server/apps/archive/archive_crop.py [hunk 1]
```diff
@@ -155,6 +156,10 @@ def _save_cropped_image(self, file_stream, file):
             crop['media'] = file_id
             crop['mime_type'] = content_type
             crop['href'] = url_for_media(file_id)
+            crop['CropTop'] = doc.get('CropTop', None)
+            crop['CropLeft'] = doc.get('CropLeft', None)
+            crop['CropRight'] = doc.get('CropRight', None)
+            crop['CropBottom'] = doc.get('CropBottom', None)
```
Comment (plamut) [inline server/apps/archive/archive_crop.py]: `None` is a bit redundant, as it is already a default _default value_ of  the `get()` method, but no harm.

**Comment by plamut** [comment]
:+1:

**Pull request merged**